### ISSUE TITLE
Talos - Improve PR Body

### ISSUE THREAD

**Issue by dr3** [BBC-archive/psammead#1755]
Blocked on https://github.com/bbc/psammead/issues/1756 which gives context to what is bumped where

**Is your feature request related to a problem? Please describe.**
The current PR body, seen here https://github.com/bbc/psammead/pull/1734 isnt great

**Describe the solution you'd like**
Update the body of the PRs that are generated to have more information. If possible include a link to the PR that caused talos to run. If possible, add the changelog version diff to the new PR description telling us what this new PR is pulling in

![image](https://user-images.githubusercontent.com/11341355/63258762-a7e04b00-c274-11e9-9e45-c63fc3faf9ea.png)
which expands to 
![image](https://user-images.githubusercontent.com/11341355/63258776-ae6ec280-c274-11e9-8796-f6dabd490094.png)

**Additional context**

Testing:
The above description is straight forward which explains what is expected.
